### ISSUE TITLE
Feature request: Scheduled backups

### ISSUE THREAD

**Issue by utzer** [moezbhatti/qksms#1917]
Add an option for scheduled backups to a predefined folder on the phone, daily should be enough, but if possible allow to set an interval.

**Comment by iamtalhaasghar** [comment]
Duplicate of #1816

**Comment by utzer** [comment]
> Duplicate of #1816

Clearly not, the mentioned feature request is a "poor man" synchronization with automatic restore, I read that before opening the request here and think #1816 is not a good idea for a feature request.

**Comment by iamtalhaasghar** [comment]
> > Duplicate of #1816
> 
> Clearly not, the mentioned feature request is a "poor man" synchronization with automatic restore, I read that before opening the request here and think #1816 is not a good idea for a feature request.

Yeah, sorry for wrong choice of words. What i meant to say was #1816 is similar to this issue. At least the auto backup thing. Isn't it?

**Comment by utzer** [comment]
> At least the auto backup thing. Isn't it?

Yes, the scheduled backup is what is the common part, I guess that makes sense.
